src/core/datatablemodule.cc

```diff
@@ -153,15 +180,28 @@ static py::oobj frame_column_data_r(const py::PKArgs& args) {
 }
 
 
-static py::PKArgs args_frame_integrity_check(
-  1, 0, 0, false, false, {"frame"}, "frame_integrity_check",
+static const char* doc_frame_integrity_check =
 R"(frame_integrity_check(frame)
 --
 
 This function performs a range of tests on the `frame` to verify
 that its internal state is consistent. It returns None on success,
 or throws an AssertionError if any problems were found.
-)");
+
+Parameters
+----------
+frame: Frame
+    A :class:`Frame` that needs to be checked for internal consistency.
+
+return: None
+
+except: AssertionError
+    An error is raised if there were any issues with the `frame`.
```

Comment:
For consistency with all the other pages I would write it as "The exception is raised".

Reply:
done